docs/index.md

```diff
@@ -1,2 +1,19 @@
 # Mobile App Development
 
+The Province of BC provides many types of mobile apps to the public. From wild life tracking, to digital ID, to employment services and more.  These apps are available for download on [Apple's App Store](https://apps.apple.com/us/developer/his-majesty-the-king-in-right-of-the/id487323275) and [Google's Play Store](https://play.google.com/store/apps/developer?id=Province+of+British+Columbia,+Canada&hl=en_CA&gl=US). The Province also creates private apps used by its employees for their job functions.
+
+The Developer Experience Team will assist you in your mobile app development journey. While we will not code, design or review your app, we will help you:
+
+* Onboard to the app stores
+* Connect with other mobile dev teams
+* Manage signing certificates
+
+Please book an appointment with us at: [Developer.Experience@gov.bc.ca](mailto:Developer.Experience@gov.bc.ca) at the start of your project. Find more information about what to expect at the meeting in our [Getting Started](getting_started.md) guide.
+
+Curious about mobile apps but not sure if you need one? We are happy to [meet and discuss](mailto:Developer.Experience@gov.bc.ca) your scenario.
```

Comment:
re: all my comments above about linking to us - I wonder if we need a "how to contact developer experience" markdown file somewhere that we can consistently link to vs spreading that info around. That way if we ever move to a ticketing system vs email it's easier to maintain the change!
don't let this block this pr, but perhaps something for the backlog if we think it's a good idea

Reply:
I've added a contact page. I've updated the links mentioned in previous comments to use this new page.